crates/cast/src/main.rs

```diff
@@ -1,20 +1,21 @@
-use crate::helpers::scarb_utils::parse_scarb_config;
 use crate::starknet_commands::account::Account;
 use crate::starknet_commands::{
     account, call::Call, declare::Declare, deploy::Deploy, invoke::Invoke, multicall::Multicall,
 };
 use anyhow::Result;
 use camino::Utf8PathBuf;
+use cast::helpers::constants::DEFAULT_ACCOUNTS_FILE;
+use cast::helpers::scarb_utils::{parse_scarb_config, CastConfig};
 use cast::{account_file_exists, get_account, get_block_id, get_provider, print_command_result};
 use clap::{Parser, Subcommand};
 
-mod helpers;
+pub mod helpers;
```

Comment:
why it needs to be `pub` ?

Reply:
Without it, the compiler complains about some functions in scarb_utils.rs not being used anywhere

https://stackoverflow.com/a/67897487

Reply:
that's odd, what functions are those? are they marked as pub?

Reply:
<img width="810" alt="image" src="https://github.com/foundry-rs/starknet-foundry/assets/65187002/d4c1ea60-1baa-45a4-82f4-af45377dfb10">

They are

Reply:
I think the root problem is mod helpers is defined twice (in main and in lib), and one of the two is not being used. It would be cool to be able to get rid of one of the two; if this is not possible let's leave your change then. sounds good?

Reply:
It worked 🚀